draft-ietf-quic-transport.md

```diff
@@ -235,12 +235,12 @@ Stream:
 
 Application:
 
-: An entity that uses QUIC to send and receive data.
+: An entity running on an endpoint that uses QUIC to send and receive data.
```

Comment:
I'm mildly negative on this.  This says too much about how the application and stack are related.